package.json

```diff
@@ -41,11 +42,13 @@
     "flow-bin": "^0.108.0",
     "flow-typed": "^2.6.1",
     "husky": "^3.0.5",
+    "jest": "^24.9.0",
     "mkdirp": "^0.5.1",
     "nodemon": "^1.19.2",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.18.2",
-    "rimraf": "^3.0.0"
+    "rimraf": "^3.0.0",
+    "supertest": "^4.0.2"
```

Comment:
I tried chakram and this is a bit different, because it's really testing as a black box, which makes it impossible to mock anything. So I think `supertest` is the best solution among the ones I know.

Reply:
I don't have opinions here. My last node project I spun up my own server in the tests and hit the end-points manually.

Reply:
yeah that's what I did with chakram: see https://github.com/firefox-devtools/profiler-server/compare/master...julienw:use-chakram?expand=1#diff-a56d2ab82e2de422f75dd163066ac919R1.

But I'm not satisfied, and also it seems that because jest doesn't run afterEach when a test fails the test server isn't stopped in that case...